From fdd1e0536d8161da2863d52a08250de7e6dbf59e Mon Sep 17 00:00:00 2001
From: Daniel Nachun <daniel.nachun@gmail.com>
Date: Sun, 10 Nov 2024 13:23:46 -0800
Subject: [PATCH] fix PKG_CFLAGS

---
 src/Makevars.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/Makevars.in b/src/Makevars.in
index 5a9ae58..fbfde38 100644
--- a/src/Makevars.in
+++ b/src/Makevars.in
@@ -1,5 +1,5 @@
 PKG_CPPFLAGS=-I. -Icommon -Ikms -Iutf8proc \
-	-D_XOPEN_SOURCE=700 -D_DEFAULT_SOURCE \
+	-D_XOPEN_SOURCE=700 -D_DEFAULT_SOURCE -D_GNU_SOURCE \
 	-DKMS_MESSAGE_ENABLE_CRYPTO -DKMS_MESSAGE_ENABLE_CRYPTO_LIBCRYPTO \
 	-DBSON_COMPILATION -DMONGOC_COMPILATION -DOPENSSL_SUPPRESS_DEPRECATED @cflags@
 
-- 
2.47.0

